schema/artist/index.js

```diff
@@ -103,11 +111,32 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      biography: {
+        type: Article.type,
+        description: 'The Artist biography article written by Artsy',
+        resolve: ({ _id }) => {
+          return positron('articles', {
+            published: true,
+            biography_for_artist_id: _id,
+            limit: 1,
+          }).then(articles => first(articles.results));
+        },
+      },
       alternate_names: {
         type: new GraphQLList(GraphQLString),
       },
       meta: Meta,
-      blurb: markdown(),
+      blurb: assign({}, markdown(), {
+        resolve: (artist, args, ast) => {
+          const blurb = markdown().resolve(artist, args, ast);
+          if (!blurb.length) {
```

Comment:
Yeah—what's going on here?

The markdown field goes out of it's way to return null instead of empty string: https://github.com/artsy/metaphysics/blob/master/schema/fields/markdown.js#L14

Also—if for some reason you did want to augment the functionality, the proper way to call this would be to use rest params.